### PR TITLE
[wmco] Enable pods eviction logging

### PR DESCRIPTION
**Pull request by jrvaldes**
Bump logging level to INFO in the OutWriter wrapper to enable
the drainerHelper to print information regarding the eviction/deletion
of pods.

Follow-up to https://github.com/openshift/windows-machine-config-operator/commit/c11fad50f01344bec63a61f7a4e56602c60f71d7

### CODE REVIEW

File: pkg/nodeconfig/nodeconfig.go [hunk 1]
```diff
@@ -66,6 +66,38 @@ type nodeConfig struct {
...
+}
+
+// OutWriter is a wrapper to enable info-level logging inside kubectl drainer implementation
+type OutWriter struct {
+	log logr.Logger
+}
+
+func (ow OutWriter) Write(p []byte) (n int, err error) {
+	// log info
+	ow.log.Info(string(p))
```
Comment (jrvaldes) [inline pkg/nodeconfig/nodeconfig.go:87]: Sample output below. There is only a minor cosmetic issue, it adds a [line break](https://github.com/openshift/windows-machine-config-operator/blob/master/vendor/k8s.io/kubectl/pkg/drain/drain.go#L292).

```
	INFO	controllers.configmap	processing	{"instances in": "windows-instances"}
	DEBUG	wc 172.31.251.202	initializing SSH connection
	INFO	nc 172.31.251.202	evicting pod default/win-webserver-wm-6756f7468f-scbm8

	INFO	nc 172.31.251.202	evicting pod default/win-webserver-wm-6756f7468f-zmnc4


```

File: pkg/nodeconfig/nodeconfig.go [hunk 2]
```diff
@@ -66,6 +66,38 @@ type nodeConfig struct {
 	additionalLabels map[string]string
 }
 
+// ErrWriter is a wrapper to enable error-level logging inside kubectl drainer implementation
```
Comment (aravindhp) [inline pkg/nodeconfig/nodeconfig.go:69]: Won't a commit message `[nodeconfig] Rearrange ErrWriter` rather than a generic refactoring message?

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Comment by jrvaldes** [comment]
/test vsphere-e2e-operator

landed in IBMCloud

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

File: pkg/nodeconfig/nodeconfig.go (hunk 2 above)
Comment (aravindhp) [reply pkg/nodeconfig/nodeconfig.go:69]: Again, `[nodeconfig] Re-arrange constructor and structs` is too generic. Think about someone reading the git log without checking the git messages itself.

**Review by aravindhp — CHANGES_REQUESTED** [review]

File: pkg/nodeconfig/nodeconfig.go (hunk 2 above)
Comment (jrvaldes) [reply pkg/nodeconfig/nodeconfig.go:69]: Right, re-worded in https://github.com/openshift/windows-machine-config-operator/pull/676/commits/bd795161d5bb057b7e27edfe429df25f3be8cc53

Comment (aravindhp) [reply pkg/nodeconfig/nodeconfig.go:69]: Sorry to be so pedantic ` [nodeconfig] Re-arrange newDrainHelper()` is not correct either. How about  `[nodeconfig] Move *Writer and associated methods`? 

>Move the drainerHelper constructor and OutWriter, ErrWriter struct
definitions to the beginning of the file, easier to find and maintain
as the file grows.

What is drainerHelper? Please fix.

File: pkg/nodeconfig/nodeconfig.go [hunk 3]
```diff
@@ -66,6 +66,38 @@ type nodeConfig struct {
...
+}
+
+func (ow OutWriter) Write(p []byte) (n int, err error) {
+	// log debug
+	ow.log.V(1).Info(string(p))
+	return len(p), nil
+}
+
+// newDrainHelper returns new drain.Helper instance
+func (nc *nodeConfig) newDrainHelper() *drain.Helper {
```
Comment (aravindhp) [inline pkg/nodeconfig/nodeconfig.go:92]: I just noticed this. `newDrainHelper()` is associated with `nodeConfig`. This lives along with the other `nodeConfig` methods.

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Review by selansen — APPROVED** [review]
lgtm

**Review by aravindhp — APPROVED** [review]
/approve

**Pull request merged**